### DISCUSSION TITLE
Horizontal lines?

### DISCUSSION THREAD

**Discussion by iconoclasthero** [Guake/guake#2251] [Q&A]
Where's the setting to make visible horizontal lines?
I can't see much at all here:

![Screenshot from 2024-06-13 12-03-17](https://github.com/Guake/guake/assets/535962/b0898f49-a2f6-4117-8ad4-f7421b5d36c1)

```
$ guake --version; uname -a
Guake Terminal: 3.10
VTE: 0.68.0
VTE runtime: 0.68.0
Gtk: 3.24.33
Linux latitude 6.5.0-35-generic #35~22.04.1-Ubuntu SMP PREEMPT_DYNAMIC Tue May  7 09:00:52 UTC 2 x86_64 x86_64 x86_64 GNU/Linux

```

**Comment by Davidy22** [comment]
It should be largely dependent on the contrast between your desktop theme and the terminal background color since we don't have a border width thickness slider and border width seems quite niche to add an extra settings option for.

**Comment by iconoclasthero** [comment]
Would it be possible, e.g., if [dark mode]; then [increase contrast of horizontal dividers]?  That could be as simple as a checkbox.

I suppose if no one else upvotes this thread then that bears out your supposition that my request is too niche to be actionable.
I run a fairly standard dark GNOME theme so it's not something that's particularly unusual, but I have no idea how to go back and modify my _entire_ desktop environment to change one display variable in guake.   It's just really inconvenient on the occasions when I split windows horizontally.  Also, there are plenty of circumstances where a horizontal slider would come in handy, e.g., on output from journalctl, several of the scripts I use for controlling mpd...
